README.MD

```diff
@@ -0,0 +1,296 @@
+BrandedToken.js
+============
+
+BrandedToken.js supports interaction with BrandedToken-contracts.
+The steps below describe the process of staking an EIP-20 token to create a BrandedToken on the value chain (assumed to be Ethereum) and minting (creating) a value-backed Utility Branded Token on the sidechain for use in applications.
```

Comment:
Markdown does not regard the single line break between basic text. If this line is meant to be a separate paragraph, it will need a blank line before it.